backend/corpora/common/utils/aws_secret.py

```diff
@@ -23,7 +24,7 @@ class AwsSecret:
     def __init__(self, name):
         self._debug("AwsSecret.__init__({})".format(name))
         self.name = name
-        self.secrets_mgr = boto3.client(service_name="secretsmanager")
+        self.secrets_mgr = boto3.client(service_name="secretsmanager", endpoint_url=os.getenv("BOTO_ENDPOINT_URL"))
```

Comment:
what does the endpoint_url do? I have an idea, but please indulge me.

Reply:
This is explained in the PR description -- basically it allows us to override default AWS service url's with a local "fake-aws" service

Reply:
I'm sorry (>_<), yeah, I should have seen that.